cmd/conduit/main.go

```diff
@@ -144,15 +162,68 @@ func runConduitInit(path string) error {
 	}
 	defer f.Close()
 
-	_, _ = f.WriteString(sampleConfig)
+	var processorList []string
+
+	if processorsFlag != "" {
+		processorList = strings.Split(processorsFlag, ",")
+	}
+
+	var importer string
+	if importerFlag == "" {
+		importerFlag = "algod"
```

Comment:
Reminder to change this to sync_algod when it gets merged.

Reply:
Stephen, can you explain?